common-content-dataflow/modules/ROOT/partials/changelog.adoc

```diff
@@ -0,0 +1,101 @@
+= Changelog
+
+This page lists changes to the Dataflow Flex Template for BigQuery and/or Google Cloud to Neo4j.
+
+You can check the deployment status of a release by running `gsutil ls gs://dataflow-templates/`.
+
+Releases are gradually deployed, some regions may be updated before others.
+To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).
+
+== Release 2025-06-10-01_RC00
+
+* Add support to see underlying Neo4j driver's Bolt logs
+
+== Release 2025-01-26-00_RC00
+
+* Add support for calendar-based version names of the Neo4j database
+
+== Release 2025-01-22-01_RC00
+
+* Switch to Java 17
+* Updated Neo4j driver version from v4.4.18 to v5.27.0
```

Comment:
Are we not yet running on 5.28.8 (i.e. the LTS)?

Reply:
It's a retroactive change-log, this date 2025-01-22, the LTS version 5.28.x did not exist yet.

We are still on v5.27.0, great observation -- maybe we should upgrade this to a later version, will hear with team